components/panel/src/index.js

```diff
@@ -70,18 +70,24 @@ const StyledBody = styled('div')({
 const Panel = ({ panelTitle, panelBody, ...props }) => (
   <StyledPanel {...props}>
     <StyledTitle>{panelTitle}</StyledTitle>
-    <StyledBody>{panelBody}</StyledBody>
+    <StyledBody>
+      {Array.isArray(panelBody)
+        ? panelBody.map((element, index) => (
+          <React.Fragment key={index}>{element}</React.Fragment>
```

Comment:
This bit looks cool tho! It might me more elegant to `import React, { Fragment } from 'React'` at the top of the file instead and then use `Fragment` instead of `React.Fragment`